src/app/cart/components/cart/cart.component.ts

```diff
@@ -32,6 +53,13 @@ export class CartComponent implements OnInit {
     this.getList();
   }
 
+  changeIdx(event: any): void {
+    this.index = this.selectOptions.findIndex((e) => e.value === event.value);
+  }
+  identify(index: number, item: IcartItem) {
```

Comment:
Add return type. Do you need an index here?

Reply:
I pass the sorting types to the pipe by this.index